src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/LspEditorFeatureDetector.cs

```diff
@@ -90,7 +109,69 @@ public bool IsLspEditorEnabled()
             _jtf.AssertUIThread();
         }
 
-        return !_lazyUseLegacyEditorTask.GetValue(_disposeTokenSource.Token);
+        _activityLog.LogInfo("Checking if LSP Editor is available");
```

Comment:
I never liked this message. Can we remove this, and put a new log message before the `return true` that just says "LSP Editor is available". It always annoyed me that to know if the editor was available, you look for the "checking" message and the lack of any other messages.

Reply:
Done